src/Apps/Artwork/Components/ArtworkBanner/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import { ContextConsumer } from "Artsy/Router"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { get } from "Utils/get"
+import { Banner } from "./Banner"
+
+import { ArtworkBanner_artwork } from "__generated__/ArtworkBanner_artwork.graphql"
+import { ArtworkBannerQuery } from "__generated__/ArtworkBannerQuery.graphql"
+
+export interface ArtworkBannerProps {
+  artwork: ArtworkBanner_artwork
+}
+
+export class ArtworkBanner extends React.Component<ArtworkBannerProps> {
+  render() {
+    const { context, partner } = this.props.artwork
+    if (!context) return null
+    // imageUrl: image for avatar
+    // initials: fallback partner initials in case image is not there.
+    // badge: in auction / at fair / in show
+    // headline:  auction / fair / show name
+    // subHeadline: partner name
+    switch (context.__typename) {
+      case "ArtworkContextAuction":
+        const auctionImage = get(partner, p => p.profile.icon.url)
+        return (
+          <Banner
+            imageUrl={auctionImage}
+            initials={partner.initials}
+            badge="In auction"
+            headline={context.name}
+            subHeadline={partner.name}
+          />
```

Comment:
Can be replaced with EntityHeader -- just need to account for the badge

Reply:
I was thinking about that too, but I believe entity header currently have different fonts/sizes for each row there and is not following the same responsive logic. Might be a good point to bring to @design team to see if there is a way we can consolidate those or they are different entities indeed.

This whole Banner component seems like it should not be specific to artwork. But don't see it used in any other designs currently.

Reply:
Yeah, looking at the designs its different enough to warrant a different component. We should make sure that the props are roughly the same just in case we might want to unify at a later point.